kernel/classes/ezscript.php

```diff
@@ -235,8 +235,17 @@ function startup()
     */
     function initialize()
     {
-        if( ob_get_length() != 0 )
-            ob_end_clean();
+        try
+        {
+            ezpKernel::instance();
```

Comment:
Hm, maybe it won't. I grabbed this from https://github.com/ezsystems/ezpublish-legacy/blob/master/kernel/private/classes/ezpcontentpublishingprocess.php#L174 without really understanding it. Do we have a test for what I'm trying to do here?